bootstrap/src/main/java/io/airlift/bootstrap/Bootstrap.java

```diff
@@ -139,9 +139,15 @@ public Bootstrap quiet()
         return this;
     }
 
+    @Deprecated
     public Bootstrap strictConfig()
     {
-        this.strictConfig = true;
+        return strictConfig(true);
+    }
+
+    public Bootstrap strictConfig(boolean strictConfig)
```

Comment:
Maybe add a method to make it "not strict", and make that deprecated also, so we can remove non-strictness in the future.

Reply:
I'll make this a separate PR